clang/test/CodeGen/X86/sse-builtins.c

```diff
@@ -6,13 +6,23 @@
 
 #include <immintrin.h>
 
+#if defined(__cplusplus) && (__cplusplus >= 201103L)
+constexpr bool match_m128(__m128 v, float x, float y, float z, float w) {
+  return v[0] == x && v[1] == y && v[2] == z && v[3] == w;
+}
+#define TEST_CONSTEXPR(...) static_assert(__VA_ARGS__)
+#else
+#define TEST_CONSTEXPR(...)
+#endif
+
```

Comment:
Put this in a .h file and include in more tests?

Reply:
Yes I think we should - I'm going to wait until I've found any more design issues but will move it into a common header soon.

Reply:
Is the plan to do that in this patch or a later one? I think it would make sense to do here given the duplication between the SSE builtin tests.